cmake/std/atdm/ctest-s-local-test-driver.sh

```diff
@@ -8,17 +8,50 @@
 # Pass in '--help' or '-h' to print help.
 #
 
-
 CTEST_S_LOCAL_DRIVER_HELP_STR="Drive builds locally and submit to CDash
 
-usage: ./ctest-s-local-test-driver.sh <build-name-keys0> <build-name-keys1> ...
+Usage: ./ctest-s-local-test-driver.sh <build-name-keys0> <build-name-keys1> ...
 
 To use, symlink into some scratch directory and then run as:
 
   ./ctest-s-local-test-driver.sh <build-name-1> <build-name-2> ...
 
 If no build names are not given, then the help message is printed.  If 'all'
-is given, then all of the builds supported for the current sytem are run.
+is given, then all of the builds supported for the current sytem are run as
+specified in the Trilinos/cmake/std/atdm/<system_name>/all_supported_builds.sh
+file.
+
+The buld names <build-name-keysi> by default must match the the name of the
+builds listed under:
+
+  Trilinos/cmake/ctest/drivers/atdm/<system_name>/drivers/
+
+By default, <build-name-keysi> is the partial build name after the build-name
+prefix to form the full build name:
+
+  ${ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX}<build-name-keysi>
+
+(where ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX is defined in the
+Trilinos/cmake/std/atdm/<system_name>/all_supported_builds.sh file) and the
+full driver script name:
+
+  Trilinos/cmake/ctest/drivers/atdm/<system_name>/drivers/
+    ${ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX}<build-name-keysi>.sh
+
+For example, ATDM_CONFIG_CTEST_S_BUILD_NAME_PREFIX=Trilinos-atdm-waterman- and
+<build-name-keysi>=cuda-9.2-release-debug gives the full build name
+'Trilinos-atdm-waterman-cuda-9.2-release-debug'.
+
+However, if env var:
+
+  ATDM_CHT_FULL_BUILD_NAME=1
```

Comment:
For my knowledge, what does "CHT" stand for?

Reply:
@rppawlo,

FYI: Changed this var name to `ATDM_CTEST_S_USE_FULL_BULD_NAME` in PR #5981 (which you just approved :-))

Thanks for taking the time to find and report issues like this!